control-plane/agents/src/bin/core/volume/specs.rs

```diff
@@ -320,18 +320,6 @@ pub(crate) async fn healthy_volume_replicas(
     }
 }
 
-/// Check if the volume state is ok for resize operation.
-pub fn vol_status_ok_for_resize(spec: &VolumeSpec) -> Result<(), SvcError> {
```

Comment:
Removed this function altogether as we are allowing expansion of published and unpublished, both types of volume.